GOVERNANCE.md

```diff
@@ -0,0 +1,5 @@
+# Node.js GitHub Bot Project Governance
+
+The Node.js GitHub Bot is overseen by the Node.js Technical Steering Committee.
+https://github.com/nodejs/TSC
+
```

Comment:
As mentioned in the chat, should we leave this open until later? I mean, we'd still need to apply for this to be true - right?